core/templates/css/.stylelintrc

```diff
@@ -22,5 +22,7 @@
         ignore: ["custom-elements"]
       }
     ],
+    "length-zero-no-unit": true
```

Comment:
@sajalasati (As per our offline discussion) I think we have already made these changes, can you please check what's going on with stylelint?

Reply:
@DubeySandeep PTAL at @Hudda's [comment](https://github.com/oppia/oppia/issues/8423#issuecomment-797452786) on the issue explaining that.